### PR TITLE
Improvements in documentation

### PR DESCRIPTION
**Pull request by victorschinaider**
Hi, Igor.
Some improvements and an fix.

I put the variable initialization in construct, the problem with the **specialCharacters** was because it was been initializated inside the **ngOnInit**, it's the first cycle of Angular after "receiving" initial values to directive, so it was generating a bug with special characters. 

So i've changed to:

```
 public constructor(private _elementRef: ElementRef, private _renderer: Renderer2 ) {
    this.modelWithSpecialCharacters = true;
 }
```

### CODE REVIEW

**Comment by victorschinaider** [comment]
And this closes the #9.

**Comment by victorschinaider** [comment]
I think #10 has to be closed too.

File: tslint.json [hunk 1]
```diff
@@ -111,7 +111,7 @@
   "rules": {
     // TypeScript Specific
     "member-access": [
-      true,
+      false,
```
Comment (NepipenkoIgor) [inline tslint.json]: @victorschinaider I think it is important use this rule and mark properties and methods with accessors

File: tslint.json [hunk 2]
```diff
@@ -132,7 +132,7 @@
         ]
       }
     ],
-    "no-any": true,
+    "no-any": false,
```
Comment (NepipenkoIgor) [inline tslint.json]: @victorschinaider I think it is important use this rule, we should use strict typescript

File: src/app/app.component.ts [hunk 3]
```diff
@@ -15,14 +14,11 @@ export class AppComponent implements OnInit {
...
   public constructor() {
-    // highlightjs.initHighlighting();
-  }
-
-  public ngOnInit(): void {
     this.form = new FormControl('30081991');
     this.cpfFormControl = new FormControl('04787954778');
   }
 
+  public ngOnInit(): void { }
```
Comment (NepipenkoIgor) [inline src/app/app.component.ts]: @victorschinaider  In my opinion we should remove empty methods.

**Comment by NepipenkoIgor** [comment]
Awesome , but i have some comments. Please look at and we can discuss  if
you don't agree with them.

2017-05-20 16:05 GMT+03:00 Victor Almeida Schinaider <
notifications@github.com>:

**Comment by victorschinaider** [comment]
Ok, no problems. I'll correct them, thanks!

**Comment by victorschinaider** [comment]
@NepipenkoIgor, corrections already done.

**Pull request merged**